contracts/contracts/interfaces/IWarpMessenger.sol

```diff
@@ -19,12 +17,7 @@ struct WarpBlockHash {
 }
 
 interface IWarpMessenger {
-    event SendWarpMessage(
-        bytes32 indexed destinationChainID,
-        address indexed destinationAddress,
-        address indexed sender,
-        bytes message
-    );
+    event SendWarpMessage(address indexed sender, bytes message);
```

Comment:
note: since we removed two index fields with this change, we could include the unsigned warp message ID if we want to.

This should be a separate change if we want it though. cc @michaelkaplan13 @ceyonur

Reply:
Yup that's what I thought after seeing this lol. It would be a great change!

Reply:
I think that'd be a great idea. To confirm, that's the ID used for querying validators for their signature of a specific message right?